testing/py-ujson/APKBUILD

```diff
@@ -2,40 +2,44 @@
 # Maintainer: Fabian Affolter <fabian@affolter-engineering.ch>
 pkgname=py-ujson
 _pkgname=ujson
-pkgver=1.33
+pkgver=1.35
 pkgrel=0
 pkgdesc="A fast Python JSON encoder and decoder"
 url="http://github.com/esnme/ultrajson"
 arch="all"
 license="BSD"
-depends="python2"
-depends_dev=""
-makedepends="python2-dev py-setuptools"
-install=""
-subpackages=""
-source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.zip"
-
-_builddir="$srcdir"/$_pkgname-$pkgver
-prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
-}
+makedepends="python2-dev py-setuptools python3-dev"
+subpackages="py3-${pkgname/py-/}:_py3 py2-${pkgname/py-/}:_py2"
+source="$_pkgname-$pkgver.tar.gz::https://github.com/esnme/ultrajson/archive/v$pkgver.tar.gz"
+builddir="$srcdir"/ultrajson-$pkgver
 
 build() {
-	cd "$_builddir"
-	python2 setup.py build || return 1
+    cd "$builddir"
+    python2 setup.py build || return 1
+    python3 setup.py build || return 1
 }
 
 package() {
-	cd "$_builddir"
-	python2 setup.py install --prefix=/usr --root="$pkgdir" || return 1
+	mkdir -p "$pkgdir"
+}
+
+_py() {
+	local python=$1
+	pkgdesc="$pkgdesc - $python"
+	install_if="$pkgname=$pkgver-r$pkgrel $python" 	
+	cd "$builddir"
+	$python setup.py install --prefix=/usr --root="$subpkgdir"
+}
+
+_py2() {
+	_py python2
```

Comment:
This may silently fail. There’s a reason why this is always the last statement in [examples on wiki](https://wiki.alpinelinux.org/wiki/APKBUILD_examples:Python).